src/api.js

```diff
@@ -1,14 +1,17 @@
+/*jslint browser*/
+/*global NULL SQLite*/
 
 
+
+// merge src/output-pre.js into src/api.js
```

Comment:
I'm not sure this comment should stay

Reply:
applied suggestion in commit 43191e7a8453b83d7bd73d77754d41f1cbaf8730